frontend/pages/beta/model/[modelId].tsx

```diff
@@ -35,7 +37,19 @@ export default function Model() {
   }
 
   if (isModelError) {
-    return <MessageAlert message={isModelError.info.message} severity='error' />
+    if (isModelError.status === 403) {
+      return (
+        <Box sx={{ height: '100vh' }}>
+          <Forbidden
+            errorMessage='If you think this is in error please contact the model owners.'
+            noMargin
+            additionalStyling={{ height: '100vh' }}
+          />
+        </Box>
+      )
+    } else {
+      return <MessageAlert message={isModelError.info.message} severity='error' />
```

Comment:
As this is a page it'll need to be using the error wrapper error component instead of a `MessageAlert`